pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -412,6 +412,58 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: PersistentVolumeControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				// PersistentVolumeBinder.volumeController.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("persistentvolumes"),
+				},
+				// PersistentVolumeBinder.syncVolume()
+				{
+					Verbs:     sets.NewString("get", "update", "create", "delete"),
+					Resources: sets.NewString("persistentvolumes"),
+				},
+				// PersistentVolumeBinder.syncVolume()
+				{
+					Verbs:     sets.NewString("update"),
+					Resources: sets.NewString("persistentvolumes/status"),
+				},
+				// PersistentVolumeBinder.claimController.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("persistentvolumeclaims"),
+				},
+				// PersistentVolumeBinder.syncClaim()
+				{
+					Verbs:     sets.NewString("get", "update", "create", "delete"),
+					Resources: sets.NewString("persistentvolumeclaims"),
+				},
+				// PersistentVolumeBinder.syncClaim()
+				{
+					Verbs:     sets.NewString("update"),
+					Resources: sets.NewString("persistentvolumeclaims/status"),
+				},
+				// PersistentVolumeRecycler.reclaimVolume() -> handleRecycle()
+				{
+					Verbs:     sets.NewString("list", "watch"),
```

Comment:
The recycler makes a Lister/Watcher to watch its recycling pod until completion.

https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util.go#L110